pkgs/development/python-modules/streaming-form-data/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, fetchFromGitHub, buildPythonPackage, isPy3k,
+cython, mypy, numpy, pytest, requests-toolbelt }:
+
+buildPythonPackage rec {
+  pname = "streaming-form-data";
+  version = "1.8.1";
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "siddhantgoel";
+    repo = "streaming-form-data";
+    rev = "v${version}";
+    sha256 = "1wnak8gwkc42ihgf0g9r7r858hxbqav2xdgqa8azid8v2ff6iq4d";
+  };
+
+  nativeBuildInputs = [ cython ];
+
+  propagatedBuildInputs = [ requests-toolbelt ];
+
+  checkInputs = [ mypy numpy pytest ];
```

Comment:
Can we use pytestCheckHook?

Reply:
pytestCheckHook doesn't seem to be able to use the correct path to the library, leading to module not found errors.